packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -548,12 +497,12 @@ def is_anonymised_directory(dirpath, ignore_private_tags=False):
 
     Returns
     -------
-    is_anonymised : bool
+    is_anonymised : `bool`
         `True` if all of the DICOM datasets read from `dirpath` have
         been anonymised, `False` otherwise.
     """
     is_anonymised = True
-    dicom_filepaths = glob(dirpath + "/**/*.dcm", recursive=True)
+    dicom_filepaths = glob(str(dirpath) + "/**/*.dcm", recursive=True)
```

Comment:
Don't change it, but just so you know `pathlib.Path` objects have quite nice glob syntax. Instead of changing paths to string we could change strings to Path and use its syntactic goodness.

Reply:
But I'm okay for the library to straddle both methods. At least for the foreseeable future.